frontend/javascripts/admin/dataset/dataset_upload_view.js

```diff
@@ -296,7 +303,8 @@ class DatasetUploadView extends React.Component<PropsWithFormAndRouter, State> {
         this.setState({ isRetrying: false, uploadProgress: resumableUpload.progress() });
       });
 
-      resumableUpload.on("fileRetry", () => {
+      resumableUpload.on("fileRetry", file => {
+        logRetryToAnalytics(file);
```

Comment:
This passes `file` but ideally it should pass the dataset's name. Is this easy to change?